### PR TITLE
utils: Nicer process start/exit logs

### PR DESCRIPTION
**Pull request by Jongy**

### CODE REVIEW

**Review by michelhe — DISMISSED** [review]
Failing tests

**Comment by Jongy** [comment]
Flakiness, I think. Merged master

File: gprofiler/utils/__init__.py [hunk 1]
```diff
@@ -118,11 +118,11 @@ def wrapper() -> Any:
 def start_process(
     cmd: Union[str, List[str]], via_staticx: bool, term_on_parent_death: bool = True, **kwargs: Any
 ) -> Popen:
-    cmd_text = " ".join(cmd) if isinstance(cmd, list) else cmd
-    logger.debug(f"Running command: ({cmd_text})")
     if isinstance(cmd, str):
         cmd = [cmd]
 
+    logger.debug("Running command", command=cmd)
```
Comment (d3dave) [inline gprofiler/utils/__init__.py:124]: Are lists in extras valid? Shouldn't it be a string value?

Comment (Jongy) [reply gprofiler/utils/__init__.py:124]: I was about to say that I'm certain it does support, but realized that I mix it with the metadata, and it seems that lists are not supported. I'll fix. thx

Comment (Jongy) [reply gprofiler/utils/__init__.py:124]: I think repr makes sense

Comment (Jongy) [reply gprofiler/utils/__init__.py:124]: Actually - it is supported, I looked at the wrong logs view. Supported fine :)

**Comment by Jongy** [comment]
It seems that tests are truly failing

**Comment by Jongy** [comment]
Now should pass

**Review by d3dave — APPROVED** [review]

**Pull request merged**